tests/test_cli.py

```diff
@@ -421,7 +429,7 @@ class MockCheckpointingClient(CommCareHqClient):
     to return mocked data.
 
     Note this client needs to be re-initialized after use."""
-    def __init__(self, mock_data):
+    def     __init__(self, mock_data):
```

Comment:
was this intentional?

Reply:
(I think it got fixed in a later commit also)

Reply:
will fix

Reply:
eb23b7f